docs/modules/getting_started/pages/installation.adoc

```diff
@@ -0,0 +1,112 @@
+= Installation
+
+On this page you will install the Stackable Operator for Apache Hive and all required dependencies.
+
+== Stackable Operators
+
+There are 2 ways to run Stackable Operators:
+
+1. Using xref:stackablectl::index.adoc[]
+2. Using Helm
+
+=== stackablectl
+
+stackablectl is the command line tool to interact with Stackable operators and our recommended way to install Operators.
+Follow the xref:stackablectl::installation.adoc[installation steps] for your platform.
+
+After you have installed stackablectl run the following command to install all Operators necessary for Apache Hive:
+
+[source,bash]
+----
+include::example$code/getting-started.sh[tag=stackablectl-install-operators]
+----
+
+The tool will show
+
+----
+[INFO ] Installing commons operator
+[INFO ] Installing secret operator
+[INFO ] Installing hive operator
+----
+
+Furthermore, we need to install a PostgreSQL and Minio instance. This is provided via the xref:stackablectl::commands/stack.adoc[Stacks] feature of stackablectl.
```

Comment:
"This feature is provided by Stack" seems like a bit of a stretch to me, the solution is hacky :sweat_smile: 

I think it's better to just install the dependencies with Helm. Airflow also needs some external dependencies, and this is how Andrew did it: https://docs.stackable.tech/airflow/nightly/getting_started/installation.html

I think this is the cleanest way of doing it (split of the dependencies into a seperate section, and just use Helm for that)
Note that then in the index.adoc you need to add Helm as a non-optional prerequisite.

Cool that you managed to do it with stackablectl though!

Reply:
Simplified the solution and split into install operators and dependecies